crates/sui-framework/packages/sui-framework/sources/config.move

```diff
@@ -181,18 +185,18 @@ module sui::config {
         data.newer_value.borrow_mut()
     }
 
-    public(package) fun borrow_most_recent<
+    public(package) fun read_newer_setting<
```

Comment:
100% fine with this name (and apologies for possibly relitigating the name...) but ... what do you think of `read_setting_next_epoch` as a possible other name for this and that kinda fits in with the other names a bit more uniformly 🫣 😬

Reply:
Switched everything to `for_next_epoch`

Reply:
Well, DenyList is still `current_epoch` and `next_epoch` but I think the `for_` helps in the Config side